src/createCycleMiddleware.js

```diff
@@ -0,0 +1,22 @@
+import {run} from '@cycle/xstream-run';
+import actionDriver, { actionListener } from './drivers/actionDriver';
+import stateDriver, { stateListener } from './drivers/stateDriver';
```

Comment:
I'm also fiddling with it... but either something like you describe, ie

```javascript
const { actionDriver, actionListener } = createActionDriver(store)
```

or leaving both drivers inlined. 
What are your thoughts? Especially in relation to testing it, that will be next